commons/commands.py

```diff
@@ -266,8 +277,14 @@
 CMD_SW_UP = "provisioner sw_upgrade --offline"
 CMD_SPACE_CHK = "df -h"
 CMD_FIND_FILE = "find /etc/cortx/ -name *.gz"
-SET_NAMESPACE = "kubectl config set-context --current --namespace={}"
+CMD_GET_ACCESS_KEY= " kubectl get pods | grep cortx-server- | cut -d ' ' -f1 | head -1 |" \
+                    " xargs -I{} kubectl exec -t {} -c cortx-rgw -- " \
```

Comment:
use $<stringname> instead of {} to use as Template not .format

Reply:
HAK8s.get_config_value() this can be used to read the cluster.conf file . 
Values can be retrieved easily from them.